toyota_prius_2010_cam.dbc

```diff
@@ -0,0 +1,203 @@
+VERSION ""
```

Comment:
can you please change the name into toyota_prius_2010_pt.dbc? so it's consistent with other dbc file names. Other than that (and the comment above), it looks good.